api/types/types.go

```diff
@@ -150,6 +150,7 @@ type Info struct {
 	Debug              bool
 	NFd                int
 	OomKillDisable     bool
+	MemSwappiness      bool
```

Comment:
@thaJeztah , Sorry for confusing comment, so what I meant was api/info was intended for systemwide info and 
the current mem-swappiness is per container I am removing above info from api/types/types.go.
Let me know if that is okay